smart_control/reinforcement_learning/agents/networks/sac_networks.py

```diff
@@ -112,7 +112,7 @@ def __init__(self, sample_spec, predefined_outer_rank=1):
     super(_TanhNormalProjectionNetworkWrapper, self).__init__(sample_spec)
     self.predefined_outer_rank = predefined_outer_rank
 
-  def call(self, inputs, network_state=(), **kwargs):
```

Comment:
unused arg `network_state`. looks like the TD3 network might need this, but not the SAC

Reply:
also, looks like the parent class `TanhNormalProjectionNetwork` from tf-agents does not use a `network_state` in its `call` function, so we confirm we can remove this unused arg from the child class